autoendpoint/src/routers/stub/router.rs

```diff
@@ -31,10 +48,12 @@ impl StubRouter {
         })
     }
 
-    /// Create Test clients for each application
+    /// Create Test clients for each application. Tests can specify which client
+    /// to use by designating the value in the `app_id` of the subscription URL. (e.g.)
```

Comment:
Was there supposed to be an example added after the description, hence the ` (e.g.)` ?